annif/backend/__init__.py

```diff
@@ -62,9 +62,12 @@ def _pav():
 
 
 def _stwfsa():
-    from . import stwfsa
+    try:
+        from . import stwfsa
 
-    return stwfsa.StwfsaBackend
+        return stwfsa.StwfsaBackend
+    except ImportError:
+        raise ValueError("STWFSA not available, cannot use stwfsa backend")
```

Comment:
Test coverage could be extended to these lines by adding a test like https://github.com/NatLibFi/Annif/blob/3451bd351273e3425a2bea683f8ac2bf3618278b/tests/test_backend.py#L86-L93